clients/web/test/spec/browserSpec.js

```diff
@@ -18,7 +18,7 @@ describe('Test the hierarchy browser modal', function () {
             if (onRestRequest) {
                 return onRestRequest.apply(this, arguments);
             }
-            return $.when(returnVal);
+            return $.Deferred().resolve(returnVal).promise();
```

Comment:
I don't understand the motivation for this change.  The jquery documentation specifically this use case for created a resolved promise.  It also follows the conventions of `Promise.resolve`.